nasa/openmct
# Sort MyItems alphabetically when building the tree #1717

Closes #1717

### All Submissions:


### Author Checklist



### How to test
* Create new objects in the MyItems folder
* Add them in an ascending or random order
* Observe that the tree is sorted alphabetically, even after new item insertion


![Screen Shot 2021-10-26 at 1 41 53 PM](https://user-images.githubusercontent.com/7417521/138941028-ec320833-0054-430c-aeff-44719d00929f.png)
